libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/routearrow/DropInRouteArrow.kt

```diff
@@ -0,0 +1,35 @@
+package com.mapbox.navigation.dropin.component.routearrow
+
+import com.mapbox.maps.MapView
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.core.MapboxNavigation
+import com.mapbox.navigation.core.lifecycle.MapboxNavigationObserver
+import com.mapbox.navigation.core.trip.session.RouteProgressObserver
+import com.mapbox.navigation.ui.maps.route.arrow.api.MapboxRouteArrowApi
+import com.mapbox.navigation.ui.maps.route.arrow.api.MapboxRouteArrowView
+import com.mapbox.navigation.ui.maps.route.arrow.model.RouteArrowOptions
+import com.mapbox.navigation.utils.internal.ifNonNull
+
+@OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
+internal class DropInRouteArrow(
```

Comment:
we're looking to get away from `DropIn*`

This looks like a `RouteArrowComponent`